### PR TITLE
Merge develop branch to feature/extended_policy_functionality

### PR DESCRIPTION
**Pull request by istoilovgithub**
Related tasks [APPLINK-32360](https://adc.luxoft.com/jira/browse/APPLINK-32360)

### CODE REVIEW

**Comment by istoilovgithub** [comment]
@LitvinenkoIra Please review

File: src/3rd_party-static/encryption/CMakeLists.txt [hunk 1]
```diff
@@ -1,14 +1,43 @@
+# Copyright (c) 2015, Ford Motor Company
```
Comment (dtrunov) [inline src/3rd_party-static/encryption/CMakeLists.txt]: @istoilovgithub Please change year to 2017

File: src/3rd_party-static/jsoncpp/CMakeLists.txt [hunk 2]
```diff
@@ -1,14 +1,42 @@
+# Copyright (c) 2016, Ford Motor Company
```
Comment (dtrunov) [inline src/3rd_party-static/jsoncpp/CMakeLists.txt]: @istoilovgithub Please chenge year to 2017

File: src/3rd_party-static/test/CMakeLists.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015, Ford Motor Company
+# Copyright (c) 2016, Ford Motor Company
```
Comment (dtrunov) [inline src/3rd_party-static/test/CMakeLists.txt]: @istoilovgithub Please chenge the year to 2017

File: src/appMain/hmi_capabilities.json [hunk 4]
```diff
@@ -349,7 +349,7 @@
 },
     "TTS":
 	{
-		"capabilities":"TEXT",
+		"capabilities":["TEXT"],
```
Comment (dtrunov) [inline src/appMain/hmi_capabilities.json]: @istoilovgithub I think you need to change method load_capabilities_from_file from hmi_capabilities_impl.cc.

File: src/3rd_party-static/encryption/CMakeLists.txt (hunk 1 above)
Comment (istoilovgithub) [inline src/3rd_party-static/encryption/CMakeLists.txt]: @dtrunov fixed with f9d3e34

File: src/3rd_party-static/jsoncpp/CMakeLists.txt (hunk 2 above)
Comment (istoilovgithub) [inline src/3rd_party-static/jsoncpp/CMakeLists.txt]: @dtrunov fixed with f9d3e34

File: src/3rd_party-static/test/CMakeLists.txt (hunk 3 above)
Comment (istoilovgithub) [inline src/3rd_party-static/test/CMakeLists.txt]: @dtrunov fixed with f9d3e34

File: src/appMain/hmi_capabilities.json (hunk 4 above)
Comment (istoilovgithub) [inline src/appMain/hmi_capabilities.json]: @dtrunov according to implementation this parameter is expected to be a single string, so I have rolled back this change came trough automatic got merge, see f9d3e34

**Pull request closed without merging**